src/utils/shared.ts

```diff
@@ -0,0 +1,99 @@
+import { Paddle, Version } from '../../types';
+import { PaddleBillingV1Info, PaddleClassicInfo, Versions } from '../constants/cdn-information';
+
+export function findScript(cdnUrl: string): HTMLScriptElement | undefined {
+  return document.querySelector<HTMLScriptElement>(`script[src="${cdnUrl}"]`) || undefined;
+}
+
+export function injectScript(src: string): HTMLScriptElement {
+  const script = document.createElement('script');
+  script.src = src;
+
+  const headOrBody = document.head || document.body;
```

Comment:
Do we want to do this in the `head` first rather than lazy load it in the body?

Reply:
I believe this is how it has always behaved, the paddle-js init is usually setup for `DOMContentLoaded` so keeping it in the head should ensure that we don't have any sort of race condition with the global `paddle` being available

Reply:
Cool.  No worries, just asking the question really 👍